pkg/kubelet/apis/config/v1beta1/defaults_test.go

```diff
@@ -0,0 +1,728 @@
+				KubeAPIQPS:                     utilpointer.Int32(1),
+				KubeAPIBurst:                   1,
+				SerializeImagePulls:            utilpointer.Bool(true),
+				EvictionHard: map[string]string{
+					"memory.available":  "1Mi",
+					"nodefs.available":  "1%",
+					"imagefs.available": "1%",
+				},
+				EvictionSoft: map[string]string{
+					"memory.available":  "2Mi",
+					"nodefs.available":  "2%",
+					"imagefs.available": "2%",
+				},
+				EvictionSoftGracePeriod: map[string]string{
+					"memory.available":  "60s",
+					"nodefs.available":  "60s",
+					"imagefs.available": "60s",
+				},
+				EvictionPressureTransitionPeriod: metav1.Duration{Duration: 60 * time.Second},
+				EvictionMaxPodGracePeriod:        1,
+				EvictionMinimumReclaim: map[string]string{
+					"imagefs.available": "1Gi",
+				},
+				PodsPerCore:                  1,
+				EnableControllerAttachDetach: utilpointer.Bool(true),
+				ProtectKernelDefaults:        true,
+				MakeIPTablesUtilChains:       utilpointer.Bool(true),
+				IPTablesMasqueradeBit:        utilpointer.Int32(1),
+				IPTablesDropBit:              utilpointer.Int32(1),
+				FeatureGates: map[string]bool{
+					"DynamicKubeletConfig": true,
+				},
+				FailSwapOn:           utilpointer.Bool(true),
+				MemorySwap:           v1beta1.MemorySwapConfiguration{SwapBehavior: "UnlimitedSwap"},
+				ContainerLogMaxSize:  "1Mi",
+				ContainerLogMaxFiles: utilpointer.Int32(1),
+				ConfigMapAndSecretChangeDetectionStrategy: v1beta1.TTLCacheChangeDetectionStrategy,
+				SystemReserved: map[string]string{
+					"memory": "1Gi",
+				},
+				KubeReserved: map[string]string{
+					"memory": "1Gi",
+				},
+				ReservedSystemCPUs:          "0,1",
+				ShowHiddenMetricsForVersion: "1.16",
+				SystemReservedCgroup:        "system-reserved-cgroup",
+				KubeReservedCgroup:          "kube-reserved-cgroup",
+				EnforceNodeAllocatable:      []string{"system-reserved"},
+				AllowedUnsafeSysctls:        []string{"kernel.msg*"},
+				VolumePluginDir:             "volume-plugin-dir",
+				ProviderID:                  "provider-id",
+				KernelMemcgNotification:     true,
+				Logging: componentbaseconfigv1alpha1.LoggingConfiguration{
+					Format:       "json",
+					Sanitization: true,
+				},
+				EnableSystemLogHandler:          utilpointer.Bool(true),
+				ShutdownGracePeriod:             metav1.Duration{Duration: 60 * time.Second},
+				ShutdownGracePeriodCriticalPods: metav1.Duration{Duration: 60 * time.Second},
+				ReservedMemory: []v1beta1.MemoryReservation{
+					{
+						NumaNode: 1,
+						Limits:   v1.ResourceList{v1.ResourceMemory: resource.MustParse("1Gi")},
+					},
+				},
+				EnableProfilingHandler:  utilpointer.Bool(true),
+				EnableDebugFlagsHandler: utilpointer.Bool(true),
+				SeccompDefault:          utilpointer.Bool(true),
+				MemoryThrottlingFactor:  utilpointer.Float64(1),
+			},
+		},
+		{
+			"NodeStatusUpdateFrequency is not zero",
```

Comment:
Thank you for your review @ehashman! When NodeStatusReportFrequency is zero and NodeStatusUpdateFrequency is not zero, NodeStatusReportFrequency is equal to NodeStatusUpdateFrequency so I'd like to cover the case.

https://github.com/kubernetes/kubernetes/blob/f560a7643314647cbc8bb5d46c6b65f5ac698953/pkg/kubelet/apis/config/v1beta1/defaults.go#L120-L127

Reply:
This test will end up needing to be updated for unrelated changes. Things get added rarely enough that this isn't a big deal, but this could have been a bit narrower to prevent knockon effects later. (non-blocking)